### PR TITLE
[Table Designer] Add validators for computed col in PK and FK

### PR DESCRIPTION
**Pull request by caohai**
- Add validators for computed col in PK and FK
- PK: pk_col
- FK: fk_col

### CODE REVIEW

File: src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerValidator.cs [hunk 1]
```diff
@@ -664,6 +666,57 @@ public List<TableDesignerIssue> Run(Dac.TableDesigner designer)
...
+            {
+                for (int j = 0; j < table.ForeignKeys.Items[i].Columns.Count; j++)
+                {
+                    var col = table.Columns.Items.First(item => item.Name == table.ForeignKeys.Items[i].Columns[j]);
+                    if (col != null && col.IsComputed && col.IsComputedPersisted != true)
+                    {
+                        errors.Add(new TableDesignerIssue()
+                        {
+                            Description = SR.ComputedColumnNeedToBePersistedInForeignKeyRuleDescription(col.Name, table.ForeignKeys.Items[i].SystemName),
+                            PropertyPath = new object[] { TablePropertyNames.ForeignKeys, i, ForeignKeyPropertyNames.ColumnMapping, j, ForeignKeyColumnMappingPropertyNames.Column }
```
Comment (caohai) [inline src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerValidator.cs:711]: this PropertyPath is set to the path of column specification in the FK, I'm not sure if this is better or should I point the user to the `IsPersisited` setting of the column directly.

File: src/Microsoft.Kusto.ServiceLayer/Localization/sr.cs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 // WARNING:
-// This file was generated by the Microsoft DataWarehouse String Resource Tool 5.0.0.0
+// This file was generated by the Microsoft DataWarehouse String Resource Tool 4.0.0.0
```
Comment (caohai) [inline src/Microsoft.Kusto.ServiceLayer/Localization/sr.cs:2]: I tried to build with latest `main` and still got this change.

File: src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerValidator.cs (hunk 1 above)
Comment (alanrenmsft) [reply src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerValidator.cs:711]: either one is fine, I don't have a preference.

File: src/Microsoft.Kusto.ServiceLayer/Localization/sr.cs (hunk 2 above)
Comment (alanrenmsft) [reply src/Microsoft.Kusto.ServiceLayer/Localization/sr.cs:2]: manually revert it for now.

**Review by alanrenmsft — APPROVED** [review]

**Comment by smartguest** [comment]
@alanrenmsft The reason for the 5.0.0.0 version of the string resource tool is that you still have net 3.1 installed and it is using that version to generate the strings. If you uninstall net 3.1 (which isn't needed anymore), it should work as expected.

**Pull request merged**